src/NUnitConsole/nunit3-console.tests/ConsoleMocks.cs

```diff
@@ -1,5 +1,5 @@
 ﻿// ***********************************************************************
```

Comment:
This is not a file move, GitHub has matched these up incorrectly.

`IConverter` was an unused file in the console project - unrelated to this PR. ConsoleMocks is a utility class to create a mock ConsoleOptions object in the test suite.